docs/html/development/architecture/anatomy.rst

```diff
@@ -24,24 +24,23 @@ The ``README``, license, ``pyproject.toml``, ``setup.py``, and so on are in the
 * ``README.rst``
 * ``setup.cfg``
 * ``setup.py``
-* ``tox.ini`` -- ``pip`` uses Tox, an automation tool, configured by this `tox.ini`_ file. ``tox.ini`` describes a few environments ``pip`` uses during development for simplifying how tests are run (complicated situation there). Example: ``tox -e -py36``. We can run tests for different versions of Python by changing “36” to “27” or similar.
-* ``.coveragerc``
+* ``noxfile.py`` -- ``pip`` uses Tox, an automation tool, configured by this `noxfile.py`_ file. ``noxfile.py`` describes a few environments ``pip`` uses during development for simplifying how tests are run (complicated situation there). Example: ``nox -s lint``, ``nox -s test-3.10``. We can run tests for different versions of Python by changing “3.10” to “3.7” or similar.
```

Comment:
We are no longer using tox. We use Nox now.

Reply:
Typo! Pushing a fix